Web Quiz Engine (Kotlin)/task/src/engine/ControllerAdvice.kt

```diff
@@ -32,4 +32,8 @@ class ControllerAdvice {
     @ExceptionHandler(QuizAuthorMismatchException::class)
     fun handleQuizAuthorMismatchException(e: QuizAuthorMismatchException) =
         ResponseEntity(mapOf("error" to e.message), HttpStatus.FORBIDDEN)
+
+    @ExceptionHandler(KotlinNullPointerException::class)
```

Comment:
What happens if you don't have this handler? Doesn't the service return a 5XX anyway if a nullpointer happens?

It feels odd to add error handling for nullpointerexception because that's never an exception we expect to happen.

Reply:
You are right, by default it will lead to a 5XX.
I got drunk on Exception handler power and went overboard.
Done. Removed